Automattic/wp-calypso
# WooExpress: Added trial upgrade confirmation card

Related to https://github.com/Automattic/wp-calypso/issues/73500

Figma: jBwlERMS350NqhJNHZzAVF-fi-98%3A33334

## Proposed Changes

This PR builds on top of https://github.com/Automattic/wp-calypso/pull/73627 and 
* Adds the illustration files
* Adds the styles for the task cards

## Testing Instructions

* Using calypso-live, access `plans/my-plan/trial-upgraded/<site-slug>`
* Check if cards' font sizes, colors, paddings, margins, hover effect, copy, and illustrations are correct

![image](https://user-images.githubusercontent.com/3801502/220743886-04d79aa5-f2b0-4a23-81b6-eed8b25f2480.png)
